upgrade/UPGRADE-v9.0.0-dev.md

```diff
@@ -1442,6 +1442,59 @@ There you can find links to upgrade notes for other versions too.
         + asset('public/**/*.*')
       ```
     - remove `getCssVersion()` function call from your templates
+    - print styles files are now included by tag `{{ encore_entry_link_tags('app') }}`
+      - but there is no 'media="print"' parameter to set link attribute
+      - so we need to upgrade our `assets/styles/frontend/*/print/main.less` and wrap all content to media query `@media print { ... your content ... }`
+    - In case you are using google fonts, we need to download font files and avoid for using `@import` from external sources. In future we can make some magics in load time performance using FontLoader etc.
+        - Example in `variables.less`
+          ```css
+            @import url('https://fonts.googleapis.com/css?family=Montserrat:400,700&display=swap&subset=latin-ext');
+          ```
+        - open link in your browser and you will see a lot of `@font-face definitions`
+        - copy these you want to use:
+            ```css
+                /* latin-ext */
+                @font-face {
+                  font-family: 'Montserrat';
+                  font-style: normal;
+                  font-weight: 400;
+                  font-display: swap;
+                  src: local('Montserrat Regular'), local('Montserrat-Regular'), url(https://fonts.gstatic.com/s/montserrat/v14/JTUSjIg1_i6t8kCHKm459Wdhyzbi.woff2) format('woff2');
+                  unicode-range: U+0100-024F, U+0259, U+1E00-1EFF, U+2020, U+20A0-20AB, U+20AD-20CF, U+2113, U+2C60-2C7F, U+A720-A7FF;
+                }
+                /* latin-ext */
+                @font-face {
+                  font-family: 'Montserrat';
+                  font-style: normal;
+                  font-weight: 700;
+                  font-display: swap;
+                  src: local('Montserrat Bold'), local('Montserrat-Bold'), url(https://fonts.gstatic.com/s/montserrat/v14/JTURjIg1_i6t8kCHKm45_dJE3gfD_u50.woff2) format('woff2');
+                  unicode-range: U+0100-024F, U+0259, U+1E00-1EFF, U+2020, U+20A0-20AB, U+20AD-20CF, U+2113, U+2C60-2C7F, U+A720-A7FF;
+                }
+            ```
+        - replace previous `@import` with it
+        - open all link `https://fonts.gstatic.com/...` in your browser and it will download font file and save them to `assets/public/fronten/fonts`. We are using [fontName][fontWeigh].[extension] filename syntax.
```

Comment:
```suggestion
        - open all links `https://fonts.gstatic.com/...` in your browser and it will download font file and save them to `assets/public/fronten/fonts`. We are using [fontName][fontWeigh].[extension] filename syntax.
```